SceneExplorer/SceneExplorerUISystem.cs

```diff
@@ -126,6 +126,8 @@ private void ToggleInputActions(bool activate, bool isEditor)
 
         private void OnToggleSceneExplorerTool(ProxyAction proxyAction, InputActionPhase inputActionPhase)
         {
+            Logging.Info($"{nameof(OnToggleSceneExplorerTool)} was called");
```

Comment:
No need imo. There's only one reason why the window may not open - you have to exit any active tool before you open inspector window otherwise inspector tool system will kill the window while switching

Reply:
Since the issue came back again after reverting the new key binding defaults I could make sure that I had no other active tools. Also, the problem was not that the window was not opening, but rather that the key binding handler methods were not called.
This is still a mysterious issue, so I suggest to keep the logs (or add something more meaningful ones?), because they can be useful if the issue also appears for other users.

Reply:
I suggest you to create a new issue, I haven't experienced such problem, and I'm working with the mod all the time when modding. I assume you cannot easily reproduce it?

Reply:
I happened only twice for me:
- First after the recent game patch update (1.1.12f1).
- Second after I reverted the default key bindings.

Let's put it aside: I removed the extra logs. Let's hope that it was specific to me and will never happen to others...